src/styles/sidebar/components/AnnotationPublishControl.scss

```diff
@@ -1,11 +1,6 @@
 @use '../../mixins/layout';
 @use '../../variables' as var;
 
-.AnnotationPublishControl {
```

Comment:
Styling for this component needs another pass to extract patterns but I didn't want to go too crazy here.